simphony/tests/test_hash.py

```diff
@@ -0,0 +1,108 @@
+# Copyright © Simphony Project Contributors
+# Licensed under the terms of the MIT License
+# (see simphony/__init__.py for details)
+
+from simphony.libraries import siepic
+from simphony.libraries import sipann
+
+
+class TestSiepicHash:
+
+    def test_hash(self):
+
+        bdc1 = siepic.BidirectionalCoupler(thickness=2.2e-7, width=5e-7)
+        bdc2 = siepic.BidirectionalCoupler(thickness=2.2e-7, width=5e-7)
+        assert (bdc1.__hash__() == bdc2.__hash__())
```

Comment:
Remove the extra layer of parenthesis on this assert and all of the rest to match the rest of the testing code.